edge/pkg/metamanager/metaserver/kubernetes/storage/sqlite/store.go

```diff
@@ -99,6 +100,18 @@ func (s *store) List(ctx context.Context, key string, opts storage.ListOptions,
 		if err != nil {
 			return err
 		}
+
+		if !opts.Predicate.Label.Empty() {
+			tmpObjLabels := unstrObj.GetLabels()
+			if tmpObjLabels == nil {
+				continue
+			}
+			var objLabels = (labels.Set)(tmpObjLabels)
+			if !opts.Predicate.Label.Matches(objLabels) {
+				continue
+			}
+		}
```

Comment:
Thanks for the fixing.
Can we use the `Matches` method directly?
```suggestion
		labelSet := labels.Set(unstrObj.GetLabels())
		if !opts.Predicate.Label.Matches(labelSet) {
			continue
		}
```

Reply:
Thanks for the suggestion. Done